config/Dockerfiles/tests.d/openstack/04_os-sg-new

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Verify the openstack security group provisioning with template data
-# distros.exclude: none
+# distros.exclude: centos7
```

Comment:
why are we excluding centos?

Reply:
@14rcole The account we are currently using to provision security groups is hitting its quota limit when we provision 3 security groups in parallel. In order to minimize the number of security groups I excluded centos7